jsapp/js/projects/projectsTable/projectsTableHeader.module.scss

```diff
@@ -55,3 +61,92 @@
     }
   }
 }
+
+// Resizers
+
+// Mechanics
+$resizer-width: sizes.$x16; // Keep this at least 10px for ease of use
+// Appearance
+$resizer-line-width: sizes.$x1;
+$resizer-line-vertical-offset: sizes.$x4;
+$resizer-area-border-radius: sizes.$x4;
+$cell-line-width: sizes.$x1;
+$cell-line-vertical-offset: sizes.$x6;
+// Theme colors
+$cell-line-color: colors.$kobo-gray-85;
+$resizer-line-color: colors.$kobo-blue;
+$resizer-area-color: colors.$kobo-light-blue;
+
+// Resizer "pad" - interactive area
+.resizer {
+  cursor: ew-resize; // col-resize <-|->   ew-resize <->
+  border-radius: $resizer-area-border-radius;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: calc(-1 * $resizer-width / 2);
+  width: $resizer-width;
+  z-index: 2; // Overlap adjacent column header and cell width indicator
+  background-color: $resizer-area-color;
+
+  // Fade transition opacity
+  opacity: 0;
+  transition: opacity 0.1s;
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+// Resizer "line"
+.resizer::after {
+  display: inline-block;
+  content: '';
+  position: absolute;
+  top: $resizer-line-vertical-offset;
+  bottom: $resizer-line-vertical-offset;
+  right: calc(($resizer-width - $resizer-line-width) / 2);
+  width: $resizer-line-width;
+  z-index: 2; // Overlap adjacent column header and cell width indicator
+  background-color: $resizer-line-color;
+  // Fade transition opacity
+  transition: opacity 0.2s;
+  opacity: 0;
+}
+.resizer:hover::after {
+  opacity: 1;
+  transition: opacity 0.2s;
+}
+
+// Cell width indicators on column hover
+.columnRoot,
+.columnRoot:hover {
+  &::before,
```

Comment:
Let's hide the `::before` line on the first visible column. Something like `.columnRoot:first-child::before {display: none !important;}` should be fine.

Reply:
Oh interesting! I included it for consistency

Reply:
✅